src/test/java/com/iexec/worker/compute/ComputeManagerServiceTests.java

```diff
@@ -105,7 +107,7 @@ void beforeEach() {
     @Test
     void shouldDownloadApp() {
         when(dockerService.getClient(taskDescription.getAppUri())).thenReturn(dockerClient);
-        when(dockerClient.pullImage(taskDescription.getAppUri())).thenReturn(true);
+        when(dockerClient.pullImage(taskDescription.getAppUri(), Duration.of(20, ChronoUnit.MILLIS))).thenReturn(true);
         Assertions.assertThat(computeManagerService.downloadApp(taskDescription)).isTrue();
```

Comment:
`shouldFailedToDownloadAppSinceTimeout`
?

Reply:
I've thought about this test and tried to implement it. But it's pretty hard to find some working test when looking at timeouts. BTW, I'm not sure this would add much value there.